src/app/space/create/workspaces/workspaces-item/workspaces-item.component.less

```diff
@@ -0,0 +1,15 @@
+@import (reference) '../../../../../assets/stylesheets/shared/osio.less';
+.margin-right-30 { margin-right: 30px; }
+.workspaces-banner {
+  .workspaces-notification { margin-top: 5px; }
+}
+.workspace-item {
```

Comment:
if these are not going to be global styles, then I would name them specific to this instance, i.e. workspaces-icon-running, etc. Each color should also use the proper PatternFly variable, and not hex codes.

If you want these to be global styles for icons, then they should be added (and defined) in `_icons.less`.